Essentials/src/com/earth2me/essentials/EssentialsUpgrade.java

```diff
@@ -65,6 +65,7 @@ public void convertKits() {
         }
 
         config.save();
+        config.load();
```

Comment:
What does this achieve? The kit won't realistically change between these two method calls.

Reply:
But if the player type _/essentials reload_ command, it does not reload the kits.yml file.

Reply:
This code is not where the configs are reloaded; this is where the one-time migration of kits from `config.yml` to `kits.yml` occurs.